04-django/01-python-intro/validate-number.py

```diff
@@ -0,0 +1,17 @@
+number= str(input())
+
+valid_numbers   =   [1,2,3,4,5,6,7,8,9,0]
+first_plus  =   "+"
+
+
+def isValidNumber(number):
+    if len(number) < 10:
+        return False
+    for index,num in enumerate(number):
+        if(str(num)==' ' and (index%3)!=0):
```

Comment:
I'm not exactly sure what you want to do here... Did you meant for checking the first character (which would be simply `index == 0`) to be special (e.g. a `+`)?